src/node.cc

```diff
@@ -3271,6 +3287,9 @@ void SetupProcessObject(Environment* env,
   env->SetMethod(process, "_setupPromises", SetupPromises);
   env->SetMethod(process, "_setupDomainUse", SetupDomainUse);
 
+  env->SetMethod(process, "_startSigintWatchdog", StartSigintWatchdog);
+  env->SetMethod(process, "_stopSigintWatchdog", StopSigintWatchdog);
```

Comment:
All other things being equal, I'd opt to put these on e.g. the util binding object (defined in `src/node_util.cc`) instead of the public `process` object.